core/build.rs

```diff
@@ -0,0 +1,13 @@
+fn main() {
+    match build_playerglobal::build_playerglobal(
+        "../".into(),
+        std::env::var("OUT_DIR").unwrap().into(),
+    ) {
+        Ok(paths) => {
+            for path in paths {
+                println!("cargo:rerun-if-changed={}", path.to_string_lossy());
+            }
+        }
+        Err(e) => panic!("Failed to build playerglobal: {:?}", e),
```

Comment:
nit: Instead of panicking, can you please `.expect()`? This way the "happy flow" will remain without an extra indentation.